openedx/core/djangoapps/credentials/tests/factories.py

```diff
@@ -0,0 +1,53 @@
+"""Factories for generating fake credentials-related data."""
+import factory
+from factory.fuzzy import FuzzyText
+
+
+class UserCredentials(factory.Factory):
+    """Factory for stubbing user credentials resources from the User Credentials
+    API (v1).
+    """
+    class Meta(object):
+        model = dict
+
+    id = factory.Sequence(lambda n: n)  # pylint: disable=invalid-name
+    username = FuzzyText(prefix='user_')
+    status = 'awarded'
+    uuid = FuzzyText(prefix='uuid_')
+    certificate_url = 'http=//credentials.edx.org/credentials/dummy-uuid'
+    credential = {}
+
+
+class ProgramCredentials(factory.Factory):
```

Comment:
Nitpick: the names of these factories should be singular, reflecting the resources returned by the API.